starfish/src/main/java/org/maxandersen/Config.java

```diff
@@ -0,0 +1,31 @@
+package org.maxandersen;
```

Comment:
We should find another base but for now at least have starfish in the name.

Reply:
done